### PR TITLE
Update to 1.20.2

### PR DESCRIPTION
**Pull request by UsainSrht**

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -2,11 +2,13 @@
...
 # Fabric Properties
 # check these on https://modmuss50.me/fabric.html
-minecraft_version=1.20.1
-yarn_mappings=1.20.1+build.1
-loader_version=0.14.21
+minecraft_version=1.20.2
+yarn_mappings=1.20.2+build.4
+loader_version=0.14.23
+#Fabric api
+fabric_version=0.90.0+1.20.2
```
Comment (Thatsmusic99) [inline gradle.properties]: How come this was added? NRS doesn't depend on the Fabric API, it also isn't added to the build.gradle.kts file.

**Review by Thatsmusic99 — COMMENTED** [review]
Knew I was missing something. :')

File: gradle.properties (hunk 1 above)
Comment (UsainSrht) [reply gradle.properties]: I don't know to be honest, just copied it from the fabric's site :)

Comment (Thatsmusic99) [reply gradle.properties]: Okie dokes, would there be any issue removing it since it's redundant?

Comment (UsainSrht) [reply gradle.properties]: go ahead

**Review by Thatsmusic99 — APPROVED** [review]

**Pull request merged**